support multiple taxonomies

cherbst opened the pull request:
Allows multiple taxonomies for groups. Adds a filter for themes/plugins to override taxonomies.

cherbst committed:
support multiple taxonomies

cherbst committed:
use last action variable to support hirachical taxonomies

cherbst committed:
fix only showing tags from first set taxonomies

imath commented:
Wow @cherbst, your pull request looks pretty nice! Thanks a lot, will look at it more carfully really soon.

cherbst committed:
only replace taxonomy slug in get_term_link

before, if the taxonomy slug also occured in the term slug,
then the term slug was also prefixed with the bp group slug

imath commented:
I'm sorry i forgot to commit a fix for BuddyPress 2.7. I'm going to commit it. So this will probably have an impact on this pull request.

imath committed:
Merge branch 'master' into allow-multiple-taxonomies

imath reviewed (changes requested):
After a quick look, i think it's important to make sure your changes are also working on network installs. And according to this line https://github.com/imath/bp-groups-taxo/pull/15/files?diff=unified#diff-8d79d3ae70c6906dd6d01c479fccbe91R112 it doesn't seem to be the case. Could you give another look?

danimalweb commented:
I'd love the see this PR merged. Can it be fixed?
